src/components/SubmissionList.js

```diff
@@ -80,10 +80,6 @@ export default function SubmissionList({ users, contributor, submissions }) {
 }
 
 SubmissionList.propTypes = {
-  contributor: PropTypes.shape({
-    displayName: PropTypes.string.isRequired,
-    photoURL: PropTypes.string.isRequired,
-  }).isRequired,
   users: PropTypes.object.isRequired,
```

Comment:
We should go through and change these general propTypes to be more specific at some point, but probably in the initial refactor, not this branch.